RoaringBitmap/src/main/java/org/roaringbitmap/RoaringBitmap.java

```diff
@@ -2140,11 +2140,15 @@ protected void lazyor(final RoaringBitmap x2) {
     }
   }
 
-  // don't forget to call repairAfterLazy() afterward
-  // important: x2 should not have been computed lazily
-  // this method is like lazyor except that it will convert
-  // the current container to a bitset
-  protected void naivelazyor(RoaringBitmap  x2) {
+  /**
+   * don't forget to call repairAfterLazy() afterward
+   * important: x2 should not have been computed lazily
+   * this method is like lazyor except that it will convert
+   * the current container to a bitset
+   *
+   * @param x2 other bitmap
+   */
+  public void naivelazyor(RoaringBitmap  x2) {
```

Comment:
@lemire If we turn `.lazyor` into a public method, should it be camelCased (i.e. `lazyOr()`)? `.lazyor` is used in many places in the project, but seemingly only in protected methods.

Reply:
@huangfeng1993 I would stress, like in `or()` the fact this is an in-place operation.

```
    /**
     * In-place bitwise OR (union) operation. The current bitmap is modified.
     *
     * @param x2 other bitmap
     */
```